internal/health_checks/health_checks.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package health_checks
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+var healthChecksLogFile = "health_checks_log.txt"
```

Comment:
```suggestion
var healthChecksLogFile = "health-checks.log"
```

Reply:
Addressed in https://github.com/GoogleCloudPlatform/ops-agent/pull/1051/commits/ee0eddfa50ef7aa064815cf903409105b17a8be6 .